src/ai/ai.cpp

```diff
@@ -1638,8 +1638,7 @@ void update_crisis_leaders(sys::state& state) {
 								for(auto s : potential_states) {
 									if(military::cb_instance_conditions_satisfied(state, par.id, target, offer_cb, state.world.state_instance_get_definition(s), dcon::national_identity_id{}, dcon::nation_id{})) {
 
-										diplomatic_message::message m;
-										memset(&m, 0, sizeof(diplomatic_message::message));
```

Comment:
Are you sure that these memsets are safe to remove? Sometimes they are used to ensure that padding and the unused parts of unions are still set to 0 so that, if they get saved to a file via memcopy, a checksum generated from that will always produce the same result (which is necessary to avoid false positive out of sync detections in multiplayer)

Reply:
Hmm I haven't been able to get local multiplayer on two clients working to test out desyncs so I'm can't guarantee it. I'll close this PR for now since it's just a file cleanup.